LIC_FILES_CHKSUM.sha256

```diff
@@ -57,12 +57,10 @@ b8514c577c1c4b46cee454d5a882b15fa411e72c5bd7f801f241591789fce61a  vendor/github.
 2d36597f7117c38b006835ae7f537487207d8ec407aa9d9980794b2030cbc067  vendor/golang.org/x/sync/LICENSE
 2d36597f7117c38b006835ae7f537487207d8ec407aa9d9980794b2030cbc067  vendor/golang.org/x/sys/LICENSE
 2d36597f7117c38b006835ae7f537487207d8ec407aa9d9980794b2030cbc067  vendor/golang.org/x/text/LICENSE
-8407b13e462f755c06db3db3a034dc1fdc9157af19c6ea8986e7d5aecf4126b3  vendor/gopkg.in/tomb.v2/LICENSE
```

Comment:
many things in the commit does not match the commit message (nitpick mode)

Reply:
This dependency disappeared when removing the dependency on `go-lib-micro/mongo/dbtest` - which I also don't mention in the commit message.
I'll try to improve on my commit descriptions.